src/base/Cellar.sol

```diff
@@ -572,6 +633,9 @@ contract Cellar is ERC4626, Ownable, Multicall {
         // `accrue` will take platform fees from 1970 to the time it is called.
         lastAccrual = uint64(block.timestamp);
 
+        // Initialize SharePriceHighWatermark
+        feeData.sharePriceHighWatermark = 10**asset.decimals();
```

Comment:
Refactor to `10**_asset.decimals()` instead, it's more gas efficient because it uses an `MLOAD` instead of an `SLOAD`.